src/EPR.Accreditation.Facade.Common/RESTservices/HttpSiteService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using EPR.Accreditation.Facade.Common.Dtos;
+using EPR.Accreditation.Facade.Common.Enums;
+using EPR.Accreditation.Facade.Common.RESTservices.Interfaces;
+using Microsoft.AspNetCore.Http;
+
+namespace EPR.Accreditation.Facade.Common.RESTservices
+{
+    public class HttpSiteService : BaseHttpService, IHttpSiteService
+    {
+        public HttpSiteService(
+            IHttpContextAccessor httpContextAccessor,
+            IHttpClientFactory httpClientFactory,
+            string baseUrl,
+            string endPointName) : base(httpContextAccessor, httpClientFactory, baseUrl, endPointName)
+        {
+        }
+
+        public async Task<Guid> CreateSite(Dtos.Site site)
+        {
+            var externalId = await Post<Guid>(site);
+            return externalId;
+        }
+
+        public async Task<Dtos.Site> GetSite(
+            Guid accreditationExternalId,
+            Guid siteExternalId)
+        {
+            return await Get<Dtos.Site>($"{accreditationExternalId}/Site/{siteExternalId}");
```

Comment:
Sites don't need an external id, unless it's an overseas site. This url is for a site, and therefore does not need an external id.... do you also need to retrieve an overseas site?

Reply:
Done.